Tests/UnitTests/Purchasing/PurchasesTests.swift

```diff
@@ -2049,12 +2083,14 @@ class PurchasesTests: XCTestCase {
         transaction.mockError = NSError.init(domain: SKErrorDomain, code: SKError.Code.paymentCancelled.rawValue)
         self.storeKitWrapper.delegate?.storeKitWrapper(self.storeKitWrapper, updatedTransaction: transaction)
 
-        expect(receivedUserCancelled).toEventually(beTrue())
-        expect(receivedError).toEventuallyNot(beNil())
-        expect(receivedError?.domain).toEventually(equal(RCPurchasesErrorCodeDomain))
-        expect(receivedError?.code).toEventually(equal(ErrorCode.purchaseCancelledError.rawValue))
-        expect(receivedUnderlyingError?.domain).toEventually(equal(SKErrorDomain))
-        expect(receivedUnderlyingError?.code).toEventually(equal(SKError.Code.paymentCancelled.rawValue))
+        expect(receivedUserCancelled).toEventuallyNot(beNil())
+
+        expect(receivedUserCancelled) == true
+        expect(receivedError).toNot(beNil())
+        expect(receivedError?.domain) == RCPurchasesErrorCodeDomain
+        expect(receivedError?.code) == ErrorCode.purchaseCancelledError.rawValue
+        expect(receivedUnderlyingError?.domain) == SKErrorDomain
+        expect(receivedUnderlyingError?.code) == SKError.Code.paymentCancelled.rawValue
```

Comment:
I've changed this to a single `toEventually`, that only checks that the value was actually set.
The rest aren't `toEventually` because if the test fails, it would be busy waiting for the condition to change, which it never would.